### ISSUE TITLE
"More Options" dropdown menu under a review sometimes gets cut off

### ISSUE THREAD

**Issue by alice-blue** [iftechfoundation/ifdb-suggestion-tracker#193]
If, on the home page or on https://ifdb.org/allnew, I click "See full review" next to one of the review previews, it takes me to a page for that review. Under the review, if I click "More Options," there is a dropdown menu, but the last two options are cut off at the bottom of the browser window:
![ifdb dropdown 1](https://user-images.githubusercontent.com/12779726/110647299-b909e900-8185-11eb-8c3b-736618558b51.jpg)

 If I go to the game's main page and click "More Options" under one of the reviews there, I can see the whole dropdown menu: 
![ifdb dropdown 2](https://user-images.githubusercontent.com/12779726/110647349-beffca00-8185-11eb-9719-5e67262de22f.jpg)

**Comment by lkcampbell** [comment]
A workaround is that you can scroll the page to see the remaining choices but I understand what you are saying. I supposed the expected behavior should be having the menu of choices appear above `More Options` if the menu runs of the bottom of the page, correct?

**Comment by lkcampbell** [comment]
> In the first example in the original post, it doesn't show the remaining choices, even when I scroll to the bottom of the browser window.

Interesting. I didn't see this behavior using Chrome Version 88.0.4324.192. What browser are you using?

**Comment by alice-blue** [comment]
I see it in Edge Version 89.0.774.50.

**Comment by alice-blue** [comment]
I realized that I _can_ scroll down and see all the options if I use the down arrow key. But if I try to drag the scroll bar down with the mouse, it doesn't work. As soon as I click the scroll bar, the menu disappears, and the scroll bar is instantly back at the bottom.

**Comment by lkcampbell** [comment]
Agreed, there is definitely different UI behavior between different browser. This is just a side tangent because the Tech team is trying to figure out a good testing strategy.

And yes, I agree, in this day and age, it is reasonable to expect a drop down menu to adjust where it draws itself if it is running off the edge of the browser window.

**Comment by alice-blue** [comment]
Or you could add more space at the bottom of the page.

**Comment by lkcampbell** [comment]
Probably is better to keep the logic in the menu control, make it responsible for drawing itself appropriately in its environment; otherwise, it fails if the menu control changes size (i.e. gets more choices added later) or if its environment changes (i.e. different browser, desktop vs. mobile, font size changes, resizing of the browser window, etc.).
